tests/test-server.js

```diff
@@ -315,4 +315,12 @@ describe('test-server', function () {
             );
         });
     });
+
+    it('Revision endpoint returns git commit hash', function (done) {
+        testRequest(getURL('/revision'), done, function (resp, result) {
+            assert.equal(resp.statusCode, 200);
+            assert.equal(result, process.env.REVISION_HASH);
```

Comment:
I get the string "undefined" back from the server, and the JS constant (?) ``undefined`` for ``process.env.REVISION_HASH``. If you force to a string, the test passes:

```
assert.equal(result, '' + process.env.REVISION_HASH);
```

Or, we could wait for PR 4399, and make sure it passes.